addons/website_blog/models/website_blog.py

```diff
@@ -129,7 +129,7 @@ def _compute_website_url(self):
 
     def _default_content(self):
         return '''
-            <p class="o_default_snippet_text">''' + _("Start writing here...") + '''</p>
+            <section class="s_text_block pt32 pb32" data-snippet="s_text_block" data-name="Text"><div class="container s_allow_columns"><p class="o_default_snippet_text">''' + _("Start writing here...") + '''</p></div></section>
```

Comment:
I don't think this is the fix we want. This also adds all the options of sections (including columns and stuff) on the blog-post content.

QSM seemed to be saying that the issue here is that the anchor option targets all children of oe_structure, which causes an option block to appear in the right bar and an overlay to show up. People Shouldn't be able to link to a random line/text block in their blog post in the first place, and adjusting the selector of the anchor option should also make the overlay and option block disappear completely.

Reply:
In fact do not care about this, I don't know why it is in the bag of ui fixes task... this is in fact handled by OME in another task ;)